src/components/__tests__/simpleDropdown.test.tsx

```diff
@@ -117,11 +118,15 @@ test('menu will go away after toggle is clicked twice', async () => {
       ))}
     />
   );
-  const toggleButton = await screen.findByText('dropdown label');
-  await user.click(toggleButton);
-  expect(screen.getByRole('menu')).toBeVisible();
-  await user.click(toggleButton);
-  expect(screen.getByRole('menu')).not.toBeVisible();
+
+  const toggleButton = await screen.getByText('dropdown label');
+
+  await act(async () => {
+    user.click(toggleButton);
+    await new Promise(r => setTimeout(r, 300)); // simulate delay
+    user.click(toggleButton);
+  });
+  expect(screen.queryByRole('menu')).not.toBeInTheDocument();
```

Comment:
required: came across this in the react testing docs. we can streamline this to rely on the framework instead of arbitrary timeouts... 
```
await act(async () => {
...
});
expect(screen.queryByRole('menu')).not.toBeInTheDocument();
```

this aligns a little better to what we did on `actionMenu.test.tsx` (you can add in the timeout option if you have issues getting this format to work)... lets do
```
await act(() => user.click(toggleButton));
await act(() => user.click(toggleButton));
await waitFor(() => expect(screen.queryByRole('menu')).not.toBeInTheDocument());
```